SocketRocket/SRWebSocket.m

```diff
@@ -570,6 +570,33 @@ - (void)_initializeStreams;
     
     _inputStream.delegate = self;
     _outputStream.delegate = self;
+    
+    NSString *networkServiceType;
+    switch (_urlRequest.networkServiceType) {
+        case NSURLNetworkServiceTypeDefault:
+            return;
+        case NSURLNetworkServiceTypeVoIP: {
+#ifdef __IPHONE_9_0
+            // voice over IP control - this service type is deprecated in favor of using PushKit for VoIP control
+            if (floor(NSFoundationVersionNumber) > NSFoundationVersionNumber_iOS_8_3) {
```

Comment:
I'm actually still in the process of examining this. In my app, I'm not sure if `PushKit` is enough to ensure that websocket data are continuously streamed after push notification is received in the background.

Yes, I'm also thinking about using a warning.